specification/metadata/metadata_generator.md

```diff
@@ -0,0 +1,37 @@
+# FOCUS Metadata Generator
+The FOCUS metadata provider logical section provides metadata about the provider of the FOCUS data. 
+
+## Logical Section: Schema Generator
+
+## Schema Definition Metadata
+
+* ### billing_generator
+    * use: Human readable name of the entity that is generating the billing data 
+    * type: STRING
+* ### provider_prefix
```

Comment:
Conceptually, the provider prefix belongs to the provider, not to the data generator: two data generators for the same provider should have the same prefix.
Maybe we should not go as far as splitting provider metadata and generator metadata, but maybe the specification could say that the provider prefix must be the same for all data generators addressing the same provider.